mmv1/products/certificatemanager/Certificate.yaml

```diff
@@ -108,7 +109,9 @@ properties:
 
       EDGE_CACHE: Certificates with scope EDGE_CACHE are special-purposed certificates,
       served from non-core Google data centers.
-      Currently allowed only for managed certificates.
+
```

Comment:
ALL_REGIONS hasn't been added yet to the [api docs](https://cloud.google.com/certificate-manager/docs/reference/certificate-manager/rest/v1/projects.locations.certificates#scope). Though, It should be visible by the time this PR got merged.